plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/expression/TestSparkExpressions.java

```diff
@@ -127,10 +127,22 @@ public void testIdentifier()
         assertParseFailure("`a`b` = 1");
     }
 
+    @Test
+    public void testArithmeticBinary()
+    {
+        assertExpressionTranslates("a = b % 1", "(\"a\" = (\"b\" % 1))");
+        assertExpressionTranslates("a = b * 1", "(\"a\" = (\"b\" * 1))");
+        assertExpressionTranslates("a = b + 1", "(\"a\" = (\"b\" + 1))");
+        assertExpressionTranslates("a = b - 1", "(\"a\" = (\"b\" - 1))");
+        assertExpressionTranslates("a = b / 1", "(\"a\" = (\"b\" / 1))");
+        assertExpressionTranslates("a = b & 1", "(\"a\" = (bitwise_and(\"b\", 1)))");
+        assertExpressionTranslates("a = b ^ 1", "(\"a\" = (bitwise_xor(\"b\", 1)))");
+    }
+
     @Test
     public void testInvalidNotBoolean()
     {
-        assertParseFailure("a + a");
+        assertParseFailure("'Spark' || 'SQL'");
```

Comment:
Is this different than what we have on Line: 173?

```java
       assertParseFailure("a = 'Spark' || 'SQL'");
```

Reply:
Different. `'Spark' || 'SQL'` returns string. `a = 'Spark' || 'SQL'` returns boolean.